app/src/main/java/com/infomaniak/mail/utils/WebViewUtils.kt

```diff
@@ -130,21 +131,26 @@ class WebViewUtils(context: Context) {
             jsBridge = JavascriptBridge(onWebViewFinishedLoading = onWebViewFinishedLoading)
         }
 
-        fun WebSettings.setupThreadWebViewSettings() {
+        private fun WebSettings.setupDefaultWebViewSettings() {
             @SuppressLint("SetJavaScriptEnabled")
             javaScriptEnabled = true
 
             loadWithOverviewMode = true
             useWideViewPort = true
 
+            cacheMode = LOAD_CACHE_ELSE_NETWORK
+        }
+
+        fun WebSettings.setupThreadWebViewSettings() {
+            setupDefaultWebViewSettings()
+
             setSupportZoom(true)
             builtInZoomControls = true
             displayZoomControls = false
         }
 
         fun WebSettings.setupNewMessageWebViewSettings() {
-            @SuppressLint("SetJavaScriptEnabled")
-            javaScriptEnabled = true
+            setupDefaultWebViewSettings()
 
             loadWithOverviewMode = true
             useWideViewPort = true
```

Comment:
These two are already defined inside `setupDefaultWebViewSettings()`. No need to define them again